corehq/apps/app_manager/models.py

```diff
@@ -2722,14 +2783,6 @@ def get_case_updates(self, case_type):
 
         return updates.union(scheduler_updates)
 
-    @property
-    def case_references(self):
```

Comment:
also had to change a test because of this: https://github.com/dimagi/commcare-hq/pull/14710/commits/4ef9ca348b124dd428669d0fbec1786904f9eb7a